content/en/releases/release-managers.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Release Managers
+type: docs
+---
```

Comment:
Is this original documentation or duplicated from elsewhere? (It looks like it'd be a better fit for https://k8s.dev/)

Reply:
@sftim -- some of the other pages link out to the Release Managers contact info, so it feels right to keep them close.

The contact info also is needed by both contributors and external consumers, which makes it a better fit for k/website.